gno.land/pkg/sdk/vm/msgs.go

```diff
@@ -79,6 +88,59 @@ func (msg MsgAddPackage) GetReceived() std.Coins {
 	return msg.Deposit
 }
 
+//----------------------------------------
+// MsgEval
+
+// MsgEval - eval a Gno Expr.
+type MsgEval struct {
+	PkgPath string         `json:"pkg_path" yaml:"pkg_path"`
+	Expr    string         `json:"expr" yaml:"expr"`
+	Caller  crypto.Address `json:"caller" yaml:"caller"`
+
+	// XXX: This field is experimental, use with care as output is likely to change
+	// Default format is machine
+	Format Format `json:"format" yaml:"format"`
+}
```

Comment:
```suggestion
	FormatOpts string `json:"format-opts" yaml:"format-opts"`
}
```

to support formatting options, such as JSONP' callback, and eventually other things later?

Reply:
see my comment above; I prefer to add it separately